tr_pud-ud-test.conllu

```diff
@@ -12166,7 +12166,7 @@
 # sent_id = w01072079
 # text = İmparator Caracalla 3. yüzyılda kısa bir süre geçerli olan yeni bir bölünme uygulamıştır.
 # text_en = By the 3rd century the emperor Caracalla made a new division which lasted only a short time.
-1	İmparator	İmparator	NOUN	NN	Number=Sing	13	nsubj	_	_
+1	İmparator	İmparator	NOUN	NN	Number=Sing	13	nsubj:outer	_	_
```

Comment:
Why is this labeled as outer subject? Isn't the actual error elsewhere? Is the clause _kısa bir süre geçerli olan yeni bir bölünme_ really subject? Isn't it rather an object clause?

Reply:
Revisiting it, I think you're right, "Imperator" should be just `nsubj` and "bölümne" definitely not `csubj` (`objd` or rather `ccomp`?)